schemas/view-schema.json

```diff
@@ -199,16 +194,44 @@
                 }
               },
               "readOnly": true
+            },
+            "focused_times": {
```

Comment:
should it be in the `focus` root?

Reply:
Maybe, I do not have a string opinion.

Reply:
I find it a bit weird to have a `focus` group for `has_focus` and not put other focus info in it.